backend/voting/admin.py

```diff
@@ -110,14 +110,21 @@ def gender(self, obj):
             "other": "🧑🏻‍🎤",
             "not_say": "⛔️",
         }
-        return emoji[obj.submission.speaker.gender]
+        speaker_gender = self._users_by_id[str(obj.speaker_id)]["gender"]
+        return emoji[speaker_gender]
 
     def view_submission(self, obj):  # pragma: no cover
         return format_html(
             '<a class="button" ' 'href="{{}}" target="_blank" >Open</a>&nbsp;',
             reverse("admin:submissions_submission_change", args=(obj.submission.id,)),
         )
 
+    def get_queryset(self, request):
```

Comment:
https://github.com/pythonitalia/pycon/pull/2477/files#

I implemented a mixin AdminUsersMixin that you can use so we don't need to copy&paste this get_queryset logic everywhere